Src/GoogleApis/Apis/[Media]/MediaApiErrorHandling.cs

```diff
@@ -0,0 +1,61 @@
+﻿/*
+Copyright 2015 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using Google.Apis.Requests;
```

Comment:
System usings first
empty line
3rd party usings
empty line
Google usings,

so please change to:

using System;
using System.Net.Http;
using System.Threading.Tasks;

using Google.Apis.Services;
using Google.Apis.Util;